# Week 4: 2D SLAM Parameters
scan_matcher_node:
  ros__parameters:
    max_iterations: 50
    convergence_threshold: 0.001

occupancy_grid_builder_node:
  ros__parameters:
    resolution: 0.05
    map_size: 100
    hit_prob: 0.7
    miss_prob: 0.4

loop_closure_detector_node:
  ros__parameters:
    correlation_threshold: 0.85
    min_travel_distance: 5.0
